blog/WorkingEfficicientlyWithJUnitInEclipseUpdate.adoc

```diff
@@ -0,0 +1,79 @@
+= Working Efficiently with JUnit in Eclipse - Update

```

Comment:
this is not following the template (see https://github.com/jbosstools/jbosstools-website/blob/master/blog/blog-title.adoc.template)

especially make sure you have the preamble values like :page-*

Also since first blog you need to to add your self to https://github.com/jbosstools/jbosstools-website/tree/master/_data/team